presence/java/src/com/genesys/Main.java

```diff
@@ -16,7 +16,7 @@
     public static void main(String[] args) {
         try {
             // Configure SDK settings
-            String accessToken = "nzdoJHtTSLVB9GRiBFddTDqQWa7NgvYrU_CNnr3AK7mMYChBNu2vqnpwFipLErCQgH4Wh5lJZb09NnSddV8TEg";
+            String accessToken = "";
```

Comment:
Oh this is related to the other Java tutorial. The tutorial actually shows setting the 'current' user's presence so adding in Client Credentials Grant wouldn't work.
Two alternatives could work:
1. Add an implicit Grant or Code Oauth authentication, but since this is Java, it might bloat the codebase.
2. Refactor the tutorial and not use user-context dependent methods.

2 is much better imo and I might just modify the code so it will update an arbitrary user given the userId.

Reply:
I agree and lets go with 2.